bin/prefix-vendor-namespace.php

```diff
@@ -57,39 +60,51 @@
 			continue 2;
 		}
 
+		$namespace_change = 0;
+		$uses_change      = 0;
+
 		$contents =	preg_replace(
 			"#^(\s*)(namespace)\s*({$quoted}[\\\\|;])#m",
 			"\$1\$2 {$new_namespace}\\\\\$3",
-			$contents
+			$contents,
+			-1,
+			$namespace_change
 		);
 
 		$contents =	preg_replace(
 			"#^(\s*)(use)\s*({$quoted}\\\\)#m",
 			"\$1\$2 {$new_namespace}\\\\\$3",
-			$contents
+			$contents,
+			-1,
+			$uses_change
 		);
 
 		if ( ! empty( $direct_replacements[ $path ] ) ) {
 			foreach ( $direct_replacements[ $path ] as $replace ) {
-				$replace  = preg_quote( $replace, '#' );
-				$contents =	preg_replace(
+				$direct_change = 0;
+				$replace       = preg_quote( $replace, '#' );
+				$contents      = preg_replace(
 					"#({$replace})#m",
 					"{$new_namespace}\\\\\$1",
-					$contents
+					$contents,
+					-1,
+					$direct_change
 				);
+				if ( 0 < $direct_change ) {
+					$uses_change += $direct_change;
+				}
 			}
 		}
+		if ( 0 === $namespace_change && 0 < $uses_change ) {
```

Comment:
Same here the counts will never be negative numbers.

```suggestion
		if ( ! $namespace_change && $uses_change ) {
```